client/src/components/SignUp/SignUpForm.jsx

```diff
@@ -0,0 +1,121 @@
+import { useState, useEffect } from 'react';
+
+import { Link } from 'react-router-dom';
+
+import { Form, Input, Button } from 'antd';
+
+import './style.css';
+
+
+const tailFormItemLayout = {
+  wrapperCol: {
+    xs: {
+      span: 24,
+      offset: 0,
+    },
+    sm: {
+      span: 16,
+      offset: 8,
+    },
+  },
+};
+
+const SignUpForm = () => {
+  const [signupData, setSignupData] = useState({});
+
+  useEffect(() => {
+    console.log(signupData);
+    return () => console.log('clean');
```

Comment:
!!!

Reply:
There is no database and routes to work on it, I will fetching process when the back-side is ready 
the console is just for testing the outputs